pwndbg/commands/telescope.py

```diff
@@ -153,10 +152,12 @@ def telescope(
         count -= address
         count = max(math.ceil(count / ptrsize), 1)
 
-    # Map of address to register string
-    reg_values: DefaultDict[int, list[str]] = collections.defaultdict(lambda: [])
+    name_values = collections.defaultdict(lambda: [])
     for reg in pwndbg.gdblib.regs.common:
-        reg_values[pwndbg.gdblib.regs[reg]].append(reg)
+        name_values[pwndbg.gdblib.regs[reg]].append(reg)
```

Comment:
its prboably better to cache `regs = pwndbg.gdblib.regs` above and use here `regs[reg]`